lib/rubocop/cop/rails/squished_sql_heredocs.rb

```diff
@@ -37,6 +37,9 @@ module Rails
       #       WHERE post_id = 1
       #   SQL
       #
+      # Some SQL syntax (e.g. PostgreSQL comments and functions) requires newlines
+      # to be preserved in order to work, thus auto-correction for this cop is not safe.
```

Comment:
This is a nitpick. Can you move this doc to the main description part?

```diff
       # Checks SQL heredocs to use `.squish`.
+      #
+      # Some SQL syntax (e.g. PostgreSQL comments and functions) requires newlines
+      # to be preserved in order to work, thus auto-correction for this cop is not safe.
       #
       # @example
```

Reply:
done